riff-raff-tools.yaml

```diff
@@ -2,7 +2,13 @@ stacks: [deploy]
 regions: [eu-west-1]
 
 deployments:
+  cloudformation:
+    type: cloud-formation
+    app: elk-data-node-rotation
+    parameters:
+      templatePath: cloudformation/cloudformation.yaml
```

Comment:
I'm a bit confused: in the teamcity script, the file is copied into `target` whereas here it is found in `cloudformation`. Is it necessary to do the copying then?

Reply:
Me too! 

As far as I can tell, the [node riffraff artefact plugin](https://github.com/guardian/node-riffraff-artefact) needs the file to be in target (although I only found this out via trial and error - see https://teamcity.gutools.co.uk/viewLog.html?buildId=192249&tab=buildResultsDiv&buildTypeId=Tools_ElasticsearchNodeRotation).

Reply:
The template path here seems to be used when uploading to S3, but I've always found this a bit magical. Perhaps @sihil can alleviate our confusion here?

Reply:
I think there is some magic being done here by `node-artefact-plugin`. Possibly https://github.com/guardian/node-riffraff-artefact/blob/master/src/riffraff-artefact.js#L113-L122 - although that seems to imply it should be in the root dir?

Ultimately (given this riff-raff.yaml) I think it needs to wind up at `<projectName>/<buildId>/cloudformation/cloudformation/cloudformation.yaml` (the deployment name, then the template path). If it deploys successfully then the magic is right, but it might well be worth demystifying it.